tests/parsers/moldev_softmax_pro/testdata/spectrum_data.txt

```diff
@@ -0,0 +1,2351 @@
+##BLOCKS= 97
```

Comment:
Lets cut down this test file size to 2 or 3 wavelengths, to keep test time down.